src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -72,4 +74,16 @@ public void testCreateResourceFail_missingField() throws Exception {
         .andDo(MockMvcResultHandlers.print())
         .andExpect(status().is4xxClientError());
   }
+
+  private void assertResourceExists(String jsonResponse) throws JsonProcessingException {
```

Comment:
maybe deserialize the response in the test proper and take a more meaningful type than String here. Links the known form of the json response more closely to where it's defined.